### DISCUSSION TITLE
Security question

### DISCUSSION THREAD

**Discussion by a2256014** [getredash/redash#6934] [Q&A] [answered]
I reported the vulnerability to security@redash.io, but I haven't received a response. Are you sure you're reporting it to this email?

**Comment by lucydodo** [comment]
Yes, you're right about reporting to that email address. I think we're missing something. I'm so sorry about that.
On a separate note, I actually just joined the team a while ago, so I don't have access to that report.
I'll let our internal channels know to check it out.

**Comment by a2256014** [comment]
감사합니다. 만약 확인되면 답잡 부탁드릴게요!

**Reply by lucydodo** [reply]
I've seen your security report, thank you for contributing to the team. 감사합니다!

**Comment by justinclift** [comment]
@a2256014 Oh, when did you report it?

**Reply by a2256014** [reply]
I sent a total of two vulnerabilities, once 10 days ago and once a day ago.

**Reply by a2256014** [reply]
If you let me know the email address I can receive it from, I can send it again.
@justinclift

**Reply by justinclift** [reply]
Hmmm, maybe send it to me directly (`justin@postgresql.org`)?

I'll distribute it privately to our team members. :smile:

**Reply by a2256014** [reply]
i send it again please check

Thanks!

**Reply by justinclift** [reply, marked as answer]
Cool, yep received two emails.  Am forwarding them to team members and replying to you at the same time. :smile:
